### ISSUE TITLE
Company's Profile 

### ISSUE THREAD

**Issue by angeljgomezc** [Developer-DAO/DAO-job-board#19]
The company profile will hold basic data about a company or startup. This data should also include links to relevant company websites if necessary. It should also have a logo and a brief description.

NOTE: Companies/startups should be on the link  = devdaojobboard.com/companies/[companyname]

**Comment by swetshaw** [comment]
It can also contain a list of available jobs in that company.

**Comment by swetshaw** [comment]
Can I take up this issue? 
I see this issue need two tasks to be done

- Page design for company profile
- Convert design to code

Is there's anything else that needs to be done?

**Comment by angeljgomezc** [comment]
> 
> 
> Can I take up this issue? I see this issue need two tasks to be done
> 
>     * Page design for company profile
> 
>     * Convert design to code
> 
> 
> Is there's anything else that needs to be done?

Yes @swetshaw! That would be great! You can start by designing the profile on Figma and dropping it here. We will review your design and let you know if you should proceed. 

A few company pages from job boards you could use as inspiration:

- [CryptocurrencyJobs](https://cryptocurrencyjobs.co/startups/toucan-protocol/)
- [Pomp Crypto Jobs](https://pompcryptojobs.com/company/1150/kraken/)
- [GetOnBoard](https://www.getonbrd.com/companies/clickit-smart-technologies)

**Comment by swetshaw** [comment]
Thanks @angeljgomezc , I am on it :)

**Comment by swetshaw** [comment]
Hi @angeljgomezc , keeping in mind the overall theme and color palette of the developer DAO website, I have come up with the design for the company profile, please have a look [here](https://www.figma.com/file/hqg2FnQCLzZBBfyeYAz23t/D_D-company-profile?node-id=1%3A3). 

Looking forward to your feedback 😃

**Comment by swetshaw** [comment]
Hi @angeljgomezc , I have a few things on my mind regarding the Developer DAO website and the company profile page, can we connect over discord anytime to discuss it? It would help in taking the next steps

**Comment by angeljgomezc** [comment]
> Hi @angeljgomezc , keeping in mind the overall theme and color palette of the developer DAO website, I have come up with the design for the company profile, please have a look [here](https://www.figma.com/file/hqg2FnQCLzZBBfyeYAz23t/D_D-company-profile?node-id=1%3A3).
> 
> Looking forward to your feedback 😃

Hey @swetshaw! This looks really cool! Yes, we can proceed with this one. Just remember to use Chakra UI for the development.

**Comment by angeljgomezc** [comment]
> Hi @angeljgomezc , I have a few things on my mind regarding the Developer DAO website and the company profile page, can we connect over discord anytime to discuss it? It would help in taking the next steps

Great! Hit me up whenever, I'm angelg on Discord :)

**Comment by frankTurtle** [comment]
> 
> 
> Hi @angeljgomezc , keeping in mind the overall theme and color palette of the developer DAO website, I have come up with the design for the company profile, please have a look [here](https://www.figma.com/file/hqg2FnQCLzZBBfyeYAz23t/D_D-company-profile?node-id=1%3A3).
> 
> Looking forward to your feedback 😃

I think it looks pretty solid 💪
As a side note, I also run a web3 job board. This is what we currently have for UI

## Desktop
![company](https://user-images.githubusercontent.com/5667044/142451679-3a83c73b-4edd-4dd9-83f9-69e63b0ade11.gif)

## Mobile
![companyMobile](https://user-images.githubusercontent.com/5667044/142451853-acecc251-151c-4f2a-9b6a-b9f14f9f9a92.gif)
